docs/get-started.md

```diff
@@ -1,42 +1,109 @@
-## Get Started
-Use the following scripts to publish chart, enable connected cluster, and add OSM Arc extension.
+# Get Started with Open Service Mesh Arc Add-on
+Use the following scripts to publish chart, enable connected cluster, and add OSM Arc add-on.
 
 ## Prerequisites
 - Running Kubernetes cluster
-- Azure CLI
+- [Install the Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli)
 
-### Set env vars
-The OSM Azure repo relies on environment variables to make it usable on your localhost. The root of the repository contains a file named `.env.example`. Copy the contents of this file into `.env`
+## Add Azure CLI extensions
+
+1. Install the latest [connectedk8s](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/quickstart-connect-cluster).
+> Download the latest [wheel file](https://github.com/Azure/azure-arc-kubernetes-preview/blob/master/extensions/connectedk8s-0.3.8-py2.py3-none-any.whl).
 ```bash
-cat .env.example > .env
+WHEEL="<location of the Azure CLI Python wheel file>"
+az extension add --source $WHEEL --yes || true
 ```
-The various environment variables are documented in the `.env.example` file itself. Modify the variables in `.env` to suite your environment.
 
-Note: Ensure the region specified in the .env file has the latest OSM arc extension available. Supported regions listed [here](https://docs.microsoft.com/en-us/azure/azure-arc/kubernetes/connect-cluster#supported-regions).
+2. Install the latest [k8s-extension](https://docs.microsoft.com/en-us/cli/azure/k8s-extension?view=azure-cli-latest).
+> Download the lastest [wheel file](https://github.com/Azure/azure-arc-kubernetes-preview/blob/master/extensions/k8s_extension_private-0.2.1b2-py3-none-any.whl).
+```bash
+WHEEL="<location of the Azure CLI Python wheel file>"
+az extension add --source $WHEEL --yes || true
+```
+
+## Enable Arc Features
+The following steps need to be run only once for a given subscription:
+
+_Optional_: set default Azure subscription for the `az` CLI:
+```bash
+az account set --subscription=<your-Azure-subscription-ID>
+```
 
+```bash
+az feature register \
+  --namespace Microsoft.KubernetesConfiguration \
+  --name sourceControlConfiguration
+```
 
-### Enable Arc Features
-This step needs to be run only once for every new subscription.
-```sh
-./scripts/enable-arc-feature.sh
+```bash
+az provider register \
+  --namespace Microsoft.ExtendedLocation
 ```
 
-### Add Arc Extension to Cluster
-This step needs to be run for each Kubernetes cluster that needs the Arc OSM extension. It will add the connectedk8s extension to the Azure CLI, enable arc on the Kubernetes cluster (installs arc agents and components), and install the osm arc extension on the Kubernetes cluster.
-```sh
-./scripts/add-extension.sh
+```bash
+az feature register \
+  --namespace Microsoft.ExtendedLocation \
+  --name CustomLocations-ppauto
```

Comment:
Minor: All these steps are not required for OSM extension and it does not use ExtendedLocation or sourceControlConfiguration feature. Only pre-reg for extensions is Microsoft.KubernetesConfiguration ResourceProvider is registered (This is already covered in #1)

az provider register \
  --namespace Microsoft.KubernetesConfiguration --wait

Reply:
This is very helpful! Will address in a follow up PR and will update our wikis.